wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Authors card

Fixes #10946 

To test:

---
- On a site with some posts, go to Stats > Period > Authors.
  - Note when switching between the Period filters, the expanded rows will reset.
- Verify the view is:

<img width="250" alt="authors_init" src="https://user-images.githubusercontent.com/1816888/52313888-83262500-296d-11e9-82f2-0cf625407ed0.png">

---
- Select an Author row.
  - Max 10 child rows will be displayed, showing the Author's posts.
  - There is no action for selecting a child row.
- Verify the view is:

<img width="250" alt="authors_expanded" src="https://user-images.githubusercontent.com/1816888/52313927-a81a9800-296d-11e9-9998-c46a2691c5f4.png">
